tap_amazon_ads/client.py

```diff
@@ -66,15 +66,20 @@ def __exit__(self, exception_type, exception_value, traceback):
     def _refresh_access_token(self) -> None:
         """Refreshes the access token."""
         LOGGER.info("Refreshing Access Token")
-        resp_json = self.post(
+        resp_json = self.make_request(
+            "POST",
             endpoint=REFRESH_URL,
-            headers={"User-Agent": self.config["user_agent"]},
+            headers={
+                "User-Agent": self.config["user_agent"],
+                "content-type": "application/x-www-form-urlencoded;charset=UTF-8"
+            },
             body={
                 "refresh_token": self.config["refresh_token"],
                 "client_id": self.config["client_id"],
                 "client_secret": self.config["client_secret"],
-                "grant_type": "refresh_token",
-            }
+                "grant_type": "refresh_token"
+            },
+            is_auth_req=False
         )
         self._access_token = resp_json["access_token"]
         expires_in_seconds = resp_json.get("expires_in", 1 * 60 * 60)
```

Comment:
```suggestion
        expires_in_seconds = resp_json.get("expires_in", 3600)
        # default expiry time is 60 minutes
```

Minor Suggestion, not necessary, but this makes it more readable.

Reply:
Addressed review comment.